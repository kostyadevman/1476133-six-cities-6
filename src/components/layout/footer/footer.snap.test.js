import React from 'react';
import {render} from '@testing-library/react';
import Footer from "./footer";

test(`Should footer render correctly`, () => {
  const {container} = render(<Footer />);
  expect(container).toMatchSnapshot();
});
