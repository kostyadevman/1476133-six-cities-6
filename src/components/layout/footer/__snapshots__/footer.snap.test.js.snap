// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should footer render correctly 1`] = `
<div>
  <footer
    class="footer container"
  >
    <a
      class="footer__logo-link"
    >
      <img
        alt="6 cities logo"
        class="footer__logo"
        height="33"
        src="img/logo.svg"
        width="64"
      />
    </a>
  </footer>
</div>
`;
